README.md

```diff
@@ -4,53 +4,80 @@ Your task is to implement the following requirements and cover at least 80% of y
 All input data will be passed in the file in CSV format.
 
 ## Fruit Store
-Let's imagine you have a fruit store. 
-There are three type of operations in your fruit store:
+Let's imagine that we have a fruit store and every day in the store there are a number of activities:
 
 ```text
+    bl - balance, remnants of fruits for yesterday
     s - supply, means you are receiving a new fruits from suppliers
     b - buy, means someone bought a fruit
     r - return, means someone who have bought the fruits now returns you it
```

Comment:
not to use one and two letters in different cases, maybe we will just change naming a bit
b - balance
s - supply
p - purchase / o -order 
r - return